noxfile.py

```diff
@@ -20,32 +96,25 @@ def python_versions() -> Union[List[str], bool]:
 
 def dependency_versions(directory: str, dependency_name: str) -> List[str]:
     """Dependenciy versions are listed in the 'tested_versions/<dependency_name>' files of the instrumentation
-    packages, and symlinked under the relevant integration tests."""
-    try:
-        with open(
-            os.path.dirname(__file__)
-            + f"/src/test/integration/{directory}/tested_versions/{dependency_name}",
-            "r",
-        ) as f:
-            all_versions = [
-                line.strip()
-                for line in f.readlines()
-                if line.strip()[0] != "!"  # We mark incompatible versions with '1'
-            ]
-            if os.getenv("TEST_PATCH_VERSIONS", "").lower() == "true":
-                print("running all versions of", dependency_name, all_versions)
-                return all_versions
-            minor_to_version: Dict[str, Version] = {}
-            for version in all_versions:
-                parsed_version = parse_version(version)
-                minor = f"{parsed_version.major}.{parsed_version.minor}"
-                if minor_to_version.get(minor, parse_version("0")) < parsed_version:
-                    minor_to_version[minor] = parsed_version
-            minor_versions = [v.public for v in minor_to_version.values()]
-            print("running minor versions of", dependency_name, minor_versions)
-            return minor_versions
-    except FileNotFoundError:
-        return []
```

Comment:
I think it can cause a bug that test will pass although they didn't run so I deleted it.